worklets/Overview.bs

```diff
@@ -262,49 +265,83 @@ is empty; it is populated when the user agent chooses to create its {{WorkletGlo
 A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
 empty; it is populated when module scripts resolved.
 
+A {{Worklet}} has a <dfn>fetched modules map</dfn>. This is a map of <a>fetch</a> requests to values
```

Comment:
Oh, yeah, that sounds good to me. In JS all maps are ordered so that was natural for me :). Maybe say things like "the list of keys of the module responses map, in insertion order".